### ISSUE TITLE
How to specify a local address for requests

### ISSUE THREAD

**Issue by aalexlee** [swift-server/async-http-client#779]
I’m looking to specify a local address when sending requests with async-http-client, but I don’t see a direct option for this. Is there a workaround or a way to achieve this functionality?

**Comment by Lukasa** [comment]
I don't believe this is supported at the moment. It's a reasonable feature though, we'd be happy to add it if someone wanted to put together a PR.
